d.configuration.md

```diff
@@ -409,14 +409,16 @@ metadata:
   labels:
     run: nginx
   name: nginx
-  namespace: one
+  namespace: limitrange
 spec:
   containers:
   - image: nginx
     name: nginx
     resources:
       requests:
         memory: "250Mi"
+      limits:
```

Comment:
why do we need that? question only mentions "250Mi' of memory

Reply:
In the limit ranges section the first question is `Create a namespace named limitrange with a LimitRange that limits pod memory to a max of 500Mi and min of 100Mi`, thus in order to create a pod without violating the policy you need to define also limits.
I tried this in the latest version of K3s (which should be a compliant distribution wrt K8s standard)

Reply:
ah, makes sense, thank you.